samples/Mvc.ControllerForMultipleAreasSample/Areas/Products/Controllers/HomeController.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace ControllerForMultipleAreasSample.Areas.Products.Controllers
```

Comment:
It might be clearer to put this somewhere else. This could just go in the root (file and namespace wise)